controllers/parser.go

```diff
@@ -126,6 +182,14 @@ func UpdateConfigFromAPIOpts(envoyConfiguration *config.EnvoyConfiguration, opts
 		}
 	}
 
+	// update validation proxy
+	proxiedServicesArray := make([]*validation.Service, 0, len(proxiedServices))
+	for _, service := range proxiedServices {
+		proxiedServicesArray = append(proxiedServicesArray, service)
+	}
+
+	proxy.UpdateServices(proxiedServicesArray)
```

Comment:
UpdateServices() could be a method in the proxy interface

Reply:
See previous comment.